src/Symfony/Component/ExpressionLanguage/Tests/LexerTest.php

```diff
@@ -127,6 +127,25 @@ public function getTokenizeData()
                 ],
                 'foo.not in [bar]',
             ],
+            [
+                [new Token('number', '0.787', 1)],
+                '0.787',
+            ],
+            [
+                [new Token('number', '0.1234', 1)],
+                '.1234',
+            ],
+            [
+                [new Token('number', '188165.1178', 1)],
```

Comment:
this assertion looks weird to me. The value of the token is actually a float, not a numeric string

Reply:
Indeed! I'll clean the existing tests.